04/LeichtySarah.Rmd

```diff
@@ -0,0 +1,15 @@
+---
+title: "Blog 4""
```

Comment:
@sleichty be careful about extra quotations in the head matter. rmarkdown can be finnicky with formatting.